### PR TITLE
fix:Keyboard goes down to toggling between Virtual Payments Address and Mobile Number

### PR DESCRIPTION
**Pull request by shaliniv16**
Fixes #303 

**Screenshots** 

videotogif_2019 02 14_05 46 24 1

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

### CODE REVIEW

**Comment by luckyman20** [comment]
@shaliniv16 Fix the Travis build errors.

**Comment by shaliniv16** [comment]
@luckyman20 please review.

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/base/BaseActivity.java [hunk 1]
```diff
@@ -56,6 +59,12 @@ public void showSwipeProgress() {
         }
     }
 
+    @Override
+    public void hideKeyboard(View view) {
+        InputMethodManager inputMethodManager = (InputMethodManager)getSystemService(Activity.INPUT_METHOD_SERVICE);
```
[the remark this answers is not in the document]
Comment (shaliniv16) [reply mifospay/src/main/java/org/mifos/mobilewallet/mifospay/base/BaseActivity.java]: @RajaVamsi11 I have already solved it.

[the remark this answers is not in the document]
Comment (RajaVamsi11) [reply mifospay/src/main/java/org/mifos/mobilewallet/mifospay/base/BaseActivity.java]: @shaliniv16 I approved the changes that I reviewed 14 days ago. Its good that you solved it.

**Comment by shiv07tiwari** [comment]
@shaliniv16 This is already fixed.

**Pull request closed without merging**